### DISCUSSION TITLE
Is there any way to add custom start_text & end_text to every forwarded message.

### DISCUSSION THREAD

**Discussion by iamazingHub** [aahnik/tgcf#236] [Q&A]
Is there any way to add custom /start_text & /end_text to every forwarded message.

Example :
original message : 
1st message) how are you? 
2nd message) What's your favorite song? 

In this case i wanna use starting text as "Hey, " & end text as " [⚠️Give Answers Properly]"

Then the forwarded message will look like 👍
1st message ) Hey, how are you? [⚠️Give Answers Properly] 
2nd message) Hey, What's your favorite song?  [⚠️Give Answers Properly]

**Comment by aahnik** [comment]
Star this repo, and watch it. This feature is planned, and keep a watch on issue #177
